site/soc/software/charts/osh/openstack-tempest/tempest.yaml

```diff
@@ -27,9 +27,15 @@ data:
       tempest:
         auth:
           tempest_roles: []
+          admin_domain_name: "Default"
+        identity:
+          admin_domain_scope: true
+          default_domain_id: "default"
         identity-feature-enabled:
           api_v2: false
           domain_specific_drivers: true
+        image:
+          http_image: "{{ cirros_test_image_url }}"
```

Comment:
while this is ok here, in the CI I think it will fail. Dont we have an already setup public image due to changes tou the chats already?